File: frontend/src/app/dto/company/company.insert.res.dto.ts
import { CompanyInsertDataResDto } from "./company.insert.data.res.dto"

export class UserInsertResDto{
    private _data! : CompanyInsertDataResDto
	private _message! : string

    public get data(): CompanyInsertDataResDto {
        return this._data;
    }

    public set data(data: CompanyInsertDataResDto) {
        this._data = data;
    }

    public get message(): string {
        return this._message;
    }

    public set message(message: string) {
        this._message = message;
    }

}

File: backend/src/main/java/com/lawencon/ticketms/dto/company/CompanyInsertResDto.java
package com.lawencon.ticketms.dto.company;

public class CompanyInsertResDto {
	private CompanyInsertDataResDto data;
	private String message;
	public CompanyInsertDataResDto getData() {
		return data;
	}
	public void setData(CompanyInsertDataResDto data) {
		this.data = data;
	}
	public String getMessage() {
		return message;
	}
	public void setMessage(String message) {
		this.message = message;
	}
}
